fakenect/fakenect.c

```diff
@@ -404,7 +414,7 @@ freenect_frame_mode freenect_find_depth_mode(freenect_resolution res, freenect_d
 
 int freenect_get_depth_mode_count()
 {
-    return 1;
+    return 2;
```

Comment:
I think there's a (pre-existing) off-by-one error with the count here (my fault when enabling the _DEPTH_MM format) so this should be bumped to 3

Reply:
Fixed.